packages/pluggableWidgets/barcode-scanner-native/src/BarcodeScanner.tsx

```diff
@@ -21,6 +21,7 @@ export class BarcodeScanner extends Component<Props> {
                 <RNCamera
                     testID={this.props.name}
                     style={{ flex: 1, justifyContent: "center", alignItems: "center" }}
+                    type={this.props.cameraType  == 'back' ? RNCamera.Constants.Type.back : RNCamera.Constants.Type.front}
```

Comment:
Could you invert the condition like this:
```
type={this.props.cameraType === 'front' ? RNCamera.Constants.Type.front : RNCamera.Constants.Type.back}
```
This is for the sake of being future proof. If the names change or have other types of cameras, this way it can fallback to back instead of front.

Also, please use three equals :)

Reply:
Good points, fixed both!